core/network/impl/peer_manager_impl.hpp

```diff
@@ -45,6 +47,8 @@ namespace kagome::network {
    public:
     static constexpr std::chrono::seconds kTimeoutForConnecting{15};
 
+    static constexpr auto kStorageActivePeersKey{"last_active_peers"};
```

Comment:
Look at [storage/predefined_keys.hpp](https://github.com/soramitsu/kagome/blob/31b25ab2d82ceb326b87b74016af137eaf350bcc/core/storage/predefined_keys.hpp)

Reply:
This is gonna be a better solution.